coalahtml/coala_html.py

```diff
@@ -29,13 +30,33 @@
 from coalib.output.printers.ListLogPrinter import ListLogPrinter
 
 
+def gen_file_set(dirpath):
+    """
+    Parse an output from `os.walk` and create a set of its contents.
+    :param dirpath:
+        Source directory.
+    :return:
+        A set of directory contents traversed recursively.
+    """
+    content_set = set()
+    for root, dir_list, file_list in os.walk(dirpath):
+        content_set.add(root)
+        for file_name in itertools.chain(dir_list, file_list):
+            content_set.add(file_name)
+
+    return content_set
+
+
 def main():
     arg_parser = get_args()
     args = arg_parser.parse_args()
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
+    main_package = gen_file_set(get_file(Constants.COALA_HTML_BASE))
+    local_package = gen_file_set(dir_path)
+    # Inclusion check
```

Comment:
No need in this comments. Also, in general, comments should explain not what are you doing, but rather, why are you doing, what you're doing. What are you doing, should be clear from the code itself.